.github/workflows/ci-sdk.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
       matrix:
         test:
         - name: Unit Tests
-          run: cargo test $(cat service-with-tests)
+          run: cargo test --all-features
```

Comment:
is this irrelevant now? (`services-with-tests`)?

Reply:
As far as I can tell, nothing is generating this file, and that's been the case for a while.